docs/use-cases.rst

```diff
@@ -43,4 +43,155 @@ Typical Use-Cases
 
 .. START SEC:FAST-RTPS-IN-ROS2
 
+.. _fastrtps_ros2:
+
+Fast-RTPS in ROS 2
+==================
+
+Fast-RTPS is the default middleware implementation in the `Open Source Robotic Fundation (OSRF) <https://www.openrobotics.org/>`_
+`Robot Operating System ROS 2 <https://index.ros.org/doc/ros2/>`_. This tutorial is an explanation of how to take full
+advantage for Fast-RTPS wide set of capabilities in a ROS 2 project.
+
+The interface between the ROS2 stack and Fast-RTPS is provided by a ROS 2 package
+`rmw_fastrtps <https://raw.githubusercontent.com/ros2/rmw_fastrtps/>`_. This packages is available in all ROS 2
+distributions, both from binaries and from sources. Event thought Fast-RTPS is the default ROS 2 middleware, it is still
+possible to explicitly specify it in two different ways:
+
+#. Exporting ``RMW_IMPLEMENTATION`` environment variable:
+
+   ::
+
+       export RMW_IMPLEMENTATION=rmw_fastrtps_cpp
+
+#. When launching your ROS 2 application:
+
+   ::
+
+       RMW_IMPLEMENTATION=rmw_fastrtps_cpp ros2 run <package> <application>
+
+``rmw_fastrtps`` actually provides not one but two different ROS 2 middleware implementations, both of them using
+Fast-RTPS as middleware layer: ``rmw_fastrtps_cpp`` and ``rmw_fastrtps_dynamic_cpp``. The main difference between the
+two is that ``rmw_fastrtps_dynamic_cpp`` uses introspection type support at run time to decide on the
+serialization/deserialization mechanism. On the other hand, ``rmw_fastrtps_cpp`` uses its own type support, which
+generates the mapping for each message type at build time.
+
+The default ROS 2 RMW implementation is ``rmw_fastrtps_cpp``. However, it is possible to select
+``rmw_fastrtps_dynamic_cpp`` using the environment variable ``RMW_IMPLEMENTATION`` as described above.
```

Comment:
I would move them here to be everything more clear and straightforward.
Replace "above" for "below" and that would be everything.

Reply:
done in fb916ea